imports/server/hooks/slack-hooks.js

```diff
@@ -27,7 +27,6 @@ class SlackHooks extends Hookset {
   }
 
   onPuzzleNoLongerSolved(puzzleId) { // eslint-disable-line no-unused-vars
-    // TODO: unarchive Slack channel
```

Comment:
lulz we were going to do channel-per-puzzle? oh to be young and naive

Reply:
yeah in hindsight I'm pretty glad we wound up doing our own chat thing instead of leaning too heavily on Slack